### PR TITLE
[apptools] Support pair wise for manifest tests in apptools

### PR DESCRIPTION
**Pull request by yunxliu**
Failure analysis:
https://crosswalk-project.org/jira/browse/XWALK-5254
https://crosswalk-project.org/jira/browse/XWALK-6020

Impacted tests(approved): new 1, update 39, delete 0
Unit test platform: [Android]
Unit test result summary: pass 38, fail 2, block 0

Impacted tests(approved): new 1, update 24, delete 0
Unit test platform: [deb]
Unit test result summary: pass 25, fail 0, block 0

Impacted tests(approved): new 1, update 22, delete 0
Unit test platform: [iOS]
Unit test result summary: pass 0, fail 23, block 0

Impacted tests(approved): new 1, update 25, delete 0
Unit test platform: [Windows]
Unit test result summary: pass 26, fail 0, block 0

BUG=https://crosswalk-project.org/jira/browse/XWALK-6019

### CODE REVIEW

File: apptools/apptools-manifest-tests/README [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+* Yun, Liu<yunx.liu@intel.com>
+
+## LICENSE
+
+Copyright (c) 2016 Intel Corporation.
+Except as noted, this software is licensed under BSD-3-Clause License.
+Please see the COPYING file for the BSD-3-Clause License.
+
+1. allpairs is seed file folder:
```
Comment (Honry) [inline apptools/apptools-manifest-tests/README]: No title for these steps, and please move "authors" and “LICENSE” to the end of README.md

File: apptools/apptools-manifest-tests/COPYING [hunk 2]
```diff
@@ -0,0 +1,24 @@
+Copyright (c) 2014 Intel Corporation.
```
Comment (Honry) [inline apptools/apptools-manifest-tests/COPYING]: 's/2014/2016'

File: apptools/apptools-manifest-tests/README [hunk 3]
```diff
@@ -0,0 +1,74 @@
+## Introduction
+
+This test suite is for wrt-manifest-android-tests
```
Comment (Honry) [inline apptools/apptools-manifest-tests/README]: Incorrect test suite name.

File: apptools/apptools-manifest-tests/README [hunk 4]
```diff
@@ -0,0 +1,166 @@
+## Introduction
```
Comment (Honry) [inline apptools/apptools-manifest-tests/README]: Please rename README as README.md

**Comment by Honry** [comment]
@haoyunfeix, PTAL.

**Comment by haoyunfeix** [comment]
@yunxliu ,could you please supply the UT result on windows platform?

**Comment by haoyunfeix** [comment]
LGTM

**Pull request merged**